lib/origen/model/exporter.rb

```diff
@@ -23,6 +23,11 @@ def export(name, options = {})
           export_wrap_with_namespaces(f, options) do |indent|
             f.puts((' ' * indent) + 'def self.extended(model)')
             indent += 2
+            # Write out packages if any
```

Comment:
I think this should be wrapped by `if top_level?` , i.e. only write out the top-level packages if the model is the top-level.
This same export method will be applied to all models in the hierarchy.